telegram/ext/_application.py

```diff
@@ -1076,6 +1082,8 @@ async def _update_fetcher(self) -> None:
                     self.create_task(
                         self.__process_update_wrapper(update),
                         update=update,
+                        name=f"Application:{self.bot.id}:process_concurrent_update_"
```

Comment:
One more thought on the `bot.id`: IIRC we do not prevent `application.create_task` from being called before `application.initialize`, right? If so, the `bot.id` would not be available yet …

Reply:
yeah but this task will only be created in `_update_fetcher`, which is called in `start()` which has a check to confirm that the app has been initialized.

Reply:
ah okay, then we're good I guess :)